polaris-react/src/components/Box/Box.scss

```diff
@@ -1,5 +1,13 @@
 @import '../../styles/common';
 
+.listReset {
+  list-style-type: none;
+  margin-block-start: 0;
+  margin-block-end: 0;
+  outline: none;
+  padding-inline-start: 0;
```

Comment:
Should we just do margin: 0; padding: 0?

Reply:
This specifically targets the browser defaults for lists.